jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleProjectPropertiesTest.java

```diff
@@ -191,21 +187,22 @@ private static Path getResource(String path) throws URISyntaxException {
 
   @Mock private TempDirectoryProvider mockTempDirectoryProvider;
   @Mock private FileResolver mockFileResolver;
-  @Mock private Jar mockJar;
-  @Mock private Project mockProject;
   @Mock private Convention mockConvention;
   @Mock private TaskContainer mockTaskContainer;
-  @Mock private PluginContainer mockPluginContainer;
   @Mock private Logger mockLogger;
-  @Mock private Gradle mockGradle;
-  @Mock private StartParameter mockStartParameter;
   @Mock private JavaPluginConvention mockJavaPluginConvention;
   @Mock private SourceSetContainer mockSourceSetContainer;
   @Mock private SourceSet mockMainSourceSet;
   @Mock private SourceSetOutput mockMainSourceSetOutput;
 
   @Mock(answer = Answers.RETURNS_DEEP_STUBS)
-  private TaskProvider<Task> mockWarTaskProvider;
+  private Project mockProject;
+
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+  private TaskProvider<Task> mockTaskProvider1;
```

Comment:
Maybe instead of `mockTaskProvider1` and `mockTaskProvider2`, they should be `mockWarTaskProvider` and `mockBootWarTaskProvider`.